tools/cli/go-whisk/whisk/wskerror.go

```diff
@@ -34,6 +34,7 @@ const NO_DISPLAY_PREFIX         bool = false
 const APPLICATION_ERR           bool = true
 const NO_APPLICATION_ERR        bool = false
 const TIMED_OUT                 bool = true
+const UNABLE_CREATE_ACTION_ERR  string = "Unable to create action '{{.name}}': {{.err}}"
```

Comment:
this change does not seem related to the rest of the commit changes.  this is probably best moved into another pr focusing on using constants as labels for all pii strings